flake.nix

```diff
@@ -125,7 +127,10 @@
             rustToolchain
             # cargo-package # Comment in when you want tests to run on every new shell
           ]
-          ++ pkgs.lib.optionals (pkgs.stdenv.isLinux) (with pkgs; [cargo-watch]); # Currently broken on macOS
+          ++ pkgs.lib.optionals pkgs.stdenv.isDarwin
```

Comment:
Ooooh, okay! Any github nixpkgs/issue you could link to? I imagine that's where you found this workaround?
Curious as to why this is necessary =)

Reply:
I am sure I can find one, if nothing else ;p